panther_bringup/README.md

```diff
@@ -63,7 +63,7 @@ Extended Kalman Filter node for more accurate odometry. For more information, re
 
 [//]: # (ROS_API_NODE_SUBSCRIBERS_START)
 
-- `~/odom/wheels` [*nav_msgs/Odometry*]: robot odometry calculated from wheels.
+- `~/odometry/wheels` [*nav_msgs/Odometry*]: robot odometry calculated from wheels.
```

Comment:
The `~/odometry..` means `/namespace` and `/node_name` are attached as a prefix. If our goal is to specify that only a `/namespace` is added -> we should use `odometry...` 

We're mistaken in most readme files in our stack, but we can fix it iteratively once a specific readme is "touched".

Reply:
Agree and should be changed in all our readme. I'm not sure if this PR is appropriate place for that changes.

Reply:
I'm not saying you should change all occurrences of this issue. Just proposed to make it right in readme files that you already edited so we can fix it successively.